docker/build.sh

```diff
@@ -122,15 +131,18 @@ if [[ "$(docker images -q ${BUILDER} 2> /dev/null)" == "" || "$(docker images -q
 
 fi
 
-if [ "$BUILD_INHOST" = "YES" ]; then
+if [ "$BUILD_INHOST" == "YES" ]; then
     # build inside the source tree
-    docker run --rm -u 1000:1000 -v $(pwd):/opt/dali ${BUILDER} /bin/bash -c "mkdir -p /opt/dali/${DALI_BUILD_DIR} && \
+    docker run --rm -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd):/opt/dali ${BUILDER} /bin/bash -c "mkdir -p /opt/dali/${DALI_BUILD_DIR} && \
                                         cd /opt/dali/${DALI_BUILD_DIR} &&         \
                                         rm -rf /opt/dali/${DALI_BUILD_DIR}/nvidia* && \
+                                        ARCH=${ARCH}                              \
+                                        WHL_PLATFORM_NAME=${WHL_PLATFORM_NAME}    \
+                                        CUDA_TARGET_ARCHS=\"${CUDA_TARGET_ARCHS}\"\
```

Comment:
No default for this? Is this intentional? Do we want to document it?

Reply:
I know it is overwritten by CMake when it's empty.

Reply:
It is intentional. If empty the defaults are used in scripts inside. The same as for most of the variables around (build_helper keeps these defaults in one place).